### ISSUE TITLE
Guide: Researcher Toolkit

### ISSUE THREAD

**Issue by daniellex0** [hackforla/UI-UX#22]
### Overview
We need to create a UX Researcher Toolkit so that UX Researchers at Hack for LA can have many helpful resources to reference in one place.

### Action Items
- [x] Research insightful and helpful UX Research resources in online articles, slack communities, and other relevant sources
   - [x] Once done, remove the "Guide: Research" label and add the "Guide: Gather Examples" label
- [ ] Gather examples of research documentation and resources some UX researchers at Hack for LA have created/used
   - [ ] Once done, remove the "Guide: Gather Examples" label and add the "Guide: Draft Guide" label
- [ ] Create a draft Toolkit document, either in markdown format in this issue or a google doc in the [ux/ui google drive]
    - [ ] Once done, remove the "Guide: Draft Guide" label and add the "Guide: Review Guide" label
- [ ] Review the Toolkit document with UX/UI community of practice
   - [ ] Once done, remove the "Guide: Review Guide" label and add the "Guide: Leadership Review" label
- [ ] Present to the Hack for LA leadership team for sign off
   - [ ] Once approved, remove the "Guide: Leadership Review" label and add the "Guide: Place Guide" label

### Resources
Update issue #15 with the name of item you are working on
[Guide Draft](https://docs.google.com/document/d/1fPBoNEv44ApGwZQHqbdIcoD3B2UI-lvI3L6BpHig2fY/edit#)

**Comment by daniellex0** [comment]
Examples of resources that can be included in the toolkit:

- Code for America Qualitative Research Guide 
- Examples of HfLA UX Research
- UX Research Soft Skills guide

**Comment by ExperimentsInHonesty** [comment]
A sample of the new guide format that we are using:https://docs.google.com/document/d/1T-h3YJ8Pp4hsfq9ouRHRnwrrHROSz0-kQz5O9VTE62U/edit#

**Comment by nagesh23nayak** [comment]
https://docs.google.com/document/d/1fPBoNEv44ApGwZQHqbdIcoD3B2UI-lvI3L6BpHig2fY/edit#

**Comment by Jane4925** [comment]
@daniellex0 The review is done! @nagesh23nayak great job! The contents are very useful!

**Comment by ExperimentsInHonesty** [comment]
@nagesh23nayak I see that there are comments from peer reviews in your document.  Are you able to keep working on this guide or do you want someone else to finish it up?

**Comment by ExperimentsInHonesty** [comment]
@Jane4925 @mklmrgn are  you finished with your peer reviews?

**Comment by Jane4925** [comment]
@ExperimentsInHonesty I did finish the peer review, see[ comment above ](https://github.com/hackforla/UI-UX/issues/22#issuecomment-845857794)

**Comment by mklmrgn** [comment]
@ExperimentsInHonesty Yep! This was completed back in April. Let me know if there's anything that needs to be done to follow up.

**Comment by ExperimentsInHonesty** [comment]
@nagesh23nayak it looks like you received the peer review on the guide.  Will you review their comments and make appropriate updates, or do you want to be unassigned from this?

**Comment by ExperimentsInHonesty** [comment]
bumping this back into your inbox just in case @nagesh23nayak it looks like you received the peer review on the guide. Will you review their comments and make appropriate updates, or do you want to be unassigned from this?

**Comment by ExperimentsInHonesty** [comment]
@nagesh23nayak  we have not heard from you, so we are going to unassign this issue.

**Comment by InbalRL** [comment]
Hi @Nazilakl 
I saw this guide (Researcher Toolkit #22 guide) on the UX/UI Backlog tab with the "Adopt me!" tag so I assigned it myself, but then I noticed you are already assigned yourself to it.
I  wanted to ask if you are working on it. If not, I would love to take it!
Thanks,
Inbal

**Comment by mklmrgn** [comment]
I was working on this a loooooong time ago.
Please take it!
ᐧ

**Comment by InbalRL** [comment]
>

**Comment by InbalRL** [comment]
@mklmrgn I would love to learn more about what you have accomplished on this guide so far. Will you be available for a 30 min meeting?

**Comment by Aparna1Gopal** [comment]
As per [Instructions](https://github.com/hackforla/UI-UX/projects/1#card-91969059) moving this to [Backlog column](https://github.com/hackforla/UI-UX/projects/1#column-12351486)
